xml/System.IO.Pipelines/FlushResult.xml

```diff
@@ -39,8 +39,8 @@
         <Parameter Name="isCompleted" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="isCanceled">To be added.</param>
-        <param name="isCompleted">To be added.</param>
+        <param name="isCanceled"><see langword="true" /> if the current <see cref="M:System.IO.Pipelines.PipeWriter.FlushAsync(System.Threading.CancellationToken)" /> operation was canceled; otherwise, <see langword="false" />.</param>
```

Comment:
```suggestion
        <param name="isCanceled"><see langword="true" /> to indicate the current <see cref="M:System.IO.Pipelines.PipeWriter.FlushAsync(System.Threading.CancellationToken)" /> operation was canceled by <see cref="M:System.IO.Pipelines.PipeWriter.CancelPendingFlush()" />; otherwise, <see langword="false" />.</param>
```

Reply:
Ok I got two very similar suggestions, I'll merge the best of both of them 😄 :

```suggestion
        <param name="isCanceled"><see langword="true" /> to indicate the current <see cref="M:System.IO.Pipelines.PipeWriter.FlushAsync(System.Threading.CancellationToken)" /> operation that produced this <see cref="T:System.IO.Pipelines.FlushResult" /> was canceled by <see cref="M:System.IO.Pipelines.PipeWriter.CancelPendingFlush()" />; otherwise, <see langword="false" />.</param>
```